src/main/java/org/mskcc/cbio/oncokb/config/Constants.java

```diff
@@ -40,6 +40,10 @@ public final class Constants {
 
     public static final String ONCOKB_TM = "OncoKB™";
 
+    public static final String VALIDATION_ERROR = "Validation failed";
```

Comment:
I meant to add the variable as constant in CreateAssesment file since it's not shared with other classes.